Source/WebCore/css/CSSProperties.json

```diff
@@ -6070,7 +6054,7 @@
                 "longhands": [
                     "background-size"
                 ],
-                "custom-parser": true
+                "parser-grammar": "<single-webkit-background-size>#"
```

Comment:
question: Does this generate any new code that might interfere with the legacy shorthand behavior? (e.g. calling `addProperty()` with `CSSPropertyBackgroundSize`)

Reply:
(This should be caught by test failures, but asking just in case)

Reply:
It shouldn't. Shorthands are skipped currently.